lib/addExcursionPointers.m

```diff
@@ -0,0 +1,132 @@
+%==========================================================================
+%This function takes in an NIDM-Results json and creates a hashmap labelling
+%objects with a pointer to their relevant excursion set. The resultant
+%hashmap takes in the id of an object from the main NIDM-Results graph and
+%outputs a number for it's corresponding ExcursionMap.
+%
+%graph - the graph from an NIDM-Results json.
+%
+%Authors: Thomas Maullin, Camille Maumet.
+%==========================================================================
+
+function labels = addExcursionPointers(graph)
+    
+    %Create an empty list for the keys and values.
+    labelValues = {};
+    labelKeys = {};
+    
```

Comment:
I see! Would it be clearer to add each (key, value) pair one by one as in this example:
```
mapObj('May') = 100.0;
```
rather than creating temporary variables to hold keys and values?